modules/vstudio/tests/vc2010/test_globals.lua

```diff
@@ -342,6 +342,56 @@ end
 	end
 
 
+	function suite.windowsTargetPlatformVersion_latest_on2019_onUWP()
+		system "uwp"
+		p.action.set("vs2019")
+		systemversion "latest"
+		prepare()
+		test.capture [[
+<PropertyGroup Label="Globals">
+	<ProjectGuid>{42B5DBC6-AE1F-903D-F75D-41E363076E92}</ProjectGuid>
+	<IgnoreWarnCompileDuplicatedFilename>true</IgnoreWarnCompileDuplicatedFilename>
```

Comment:
This isn't something that I added for this PR but here is the reasoning for it: https://github.com/premake/premake-core/blob/7d241a6d98ce865745453929e340727a69f6ed62/modules/vstudio/vs2010_vcxproj.lua#L2055-L2064

It might not be an issue for newer versions of VS, but I'm not sure it matters if we emit it or not.